crates/rspack_error/tests/out_of_order/multi_json_syntax_error/fixtures__multi_json_syntax_error.snap

```diff
@@ -2,18 +2,15 @@
 source: crates/rspack_error/tests/fixtures.rs
 expression: multi_json_syntax_error
 ---
-error[json]: Json parsing error
-  ┌─ tests/out_of_order/multi_json_syntax_error/a.json:3:1
-  │
-1 │ {
-2 │   "a"
-3 │ }
-  │ ^ Unexpected character }
-
-error[json]: Json parsing error
-  ┌─ tests/out_of_order/multi_json_syntax_error/b.json:1:10
-  │
-1 │ "测试utf8\"
-  │            ^ Unexpected end of JSON
-
```

Comment:
this diagnostic is missing.

Reply:
This seems caused by span provided to label is out of range of the original text.

Reply:
Unfortunately, yes. After changing this line to `.len() - 1`, It works.
https://github.com/web-infra-dev/rspack/blob/2cbe062791edfc837f9a9c324a3f47d56c4edc5a/crates/rspack_plugin_json/src/lib.rs#L78
```
    7     7 │   × Error[json]: Json parsing error
    8     8 │    ╭─[tests/out_of_order/multi_json_syntax_error/b.json:1:1]
    9     9 │  1 │ "测试utf8\"
         10 │+   ·           ▲
         11 │+   ·           ╰── Unexpected end of JSON
   10    12 │    ╰────
```

Reply:
lgtm